client/src/utilities/feedback.js

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable no-undef, no-use-before-define, no-underscore-dangle */
 
-let feedbackInited;
 
 export const init = (state) => {
-  if (feedbackInited || !state.profile) {
+  if (!state || !state.profile) {
     return;
   }
-  feedbackInited = true;
```

Comment:
My understanding of the flag `feedbackInitiated` is for calling `window.bugyard` only once as the `init` function is hooked to the store changes:
* https://github.com/akvo/akvo-lumen/blob/2bb84614edb64c3a9c233a95830784a41e942409/client/src/app.jsx#L32
* https://redux.js.org/api/store/#subscribelistener

Reply:
yep, you're right but user details not working yet so was trying things to know more about the problem